README.md

```diff
@@ -5,23 +5,52 @@
 
 ## Run the tutorial on your computer
 
-To run the tutorial locally, setup the environment with [conda](https://docs.conda.io/en/latest/miniconda.html) (or [mamba](https://mamba.readthedocs.io/en/latest/installation.html)):
+### 0. Prerequisites
+
+To run the tutorial locally, you should first install [conda](https://docs.conda.io/en/latest/miniconda.html) (or [mamba](https://mamba.readthedocs.io/en/latest/installation.html)).
+
+It is also suggested that you have a recent version of `git`. Check out [how to install `git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your operating system.
+
+### 1. Download the material
+
+Go to the directory on your machine where you want to download the material and clone the repository:
 
 ```console
-conda env create -f environment.yml
+git clone https://github.com/empa-scientific-it/python-tutorial
 ```
 
+Alternatively, you can manually download a ZIP archive with the latest version of the material:
+
+[Download ZIP archive](https://github.com/empa-scientific-it/python-tutorial/archive/refs/heads/main.zip)
+
+Extract the archive in a directory of your choice.
+
+### 2. Create a dedicated environment
+
```

Comment:
Maybe add something like:
```suggestion

Enter the tutorial folder:
```console
$ cd path/to/python-tutorial
```
```

Reply:
I'll apply this in another commit: it's not clear if the `suggestion` is  messing up with the code block

Reply:
I'll apply this in another commit: it's not clear if the `suggestion` is  messing up with the code block